src/journal/journald-stream.c

```diff
@@ -246,7 +246,7 @@ static int stdout_stream_log(StdoutStream *s, const char *p) {
                 return 0;
 
         if (s->forward_to_syslog || s->server->forward_to_syslog)
-                server_forward_syslog(s->server, syslog_fixup_facility(priority), s->identifier, p, &s->ucred, NULL);
+                server_forward_syslog(s->server, syslog_fixup_facility(priority), s->identifier, p, strlen(p), &s->ucred, NULL);
```

Comment:
Should we use `strlen`?

(note `{iov_base="HEY", iov_len=3}`. should be `{iov_base="HEY\0HO", iov_len=6}`)
```sh
274   08:10:28.891967 read(19<UNIX:[12193]>, "bash\na.service\n30\n1\n0\n0\n0\nHEY\0HO", 2048) = 32 <0.000315>
274   08:10:29.443311 sendmsg(3<UNIX:[7592]>, {msg_name={sa_family=AF_UNIX, sun_path="/run/systemd/journal/syslog"}, msg_namelen=29, msg_iov=[{iov_base="<30>", iov_len=4}, {iov_base="Jan  8 08:10:29 ", iov_len=16}, {iov_base="bash", iov_len=4}, {iov_base="[767]: ", iov_len=7}, {iov_base="HEY", iov_len=3}], msg_iovlen=5, msg_control=[{cmsg_len=28, cmsg_level=SOL_SOCKET, cmsg_type=SCM_CREDENTIALS, cmsg_data={pid=767, uid=0, gid=0}}], msg_controllen=28, msg_flags=0}, MSG_NOSIGNAL)
```